src/templates/withDefaultContext.ts

```diff
@@ -1,9 +1,6 @@
+import intl = require('intl');
```

Comment:
❗️ import + require. What is this sorcery?! :O

Reply:
Ive stumbled upon it only here https://palantir.github.io/tslint/rules/no-var-requires/, but cannot find any official spec for this

Reply:
Found it! Its a Typescript specific feature https://www.typescriptlang.org/docs/handbook/modules.html. Section export = and import = require(). 💡

Reply:
```suggestion
import * as intl from 'intl';
```
This is not classy :tophat: and unaccaptable. It is excused only when usign `export = ` sytnax for class/interface/type to make TS happy, as stated in the docs.